webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue

```diff
@@ -23,49 +27,27 @@
         </option>
       </select>
     </v-scale-transition>
+    <v-scale-transition >
+      <v-btn
+          v-show="isMobile && showMobileFilter || !isMobile"
+          class="btn px-2 btn-primary"
+          outlined
+          @click="openPeopleAdvancedFilterDrawer()"
+      >
+        <i class="uiIcon uiIcon24x24 settingsIcon primary--text mr-1"></i>
+        <span class="d-none font-weight-regular caption d-sm-inline mr-1">
+            {{ $t('profile.label.search.openSearch') }}
+            <span v-if="advancedFilterCountDisplay > 0">{{`(${advancedFilterCountDisplay})`}}</span>
+          </span>
```

Comment:
1. Indentation
2. No need to a second span
3. use a computed to display a single value here instead of making conditions on extra DOM display

```suggestion
        <span class="d-none font-weight-regular caption d-sm-inline mr-1">
            {{ $t('profile.label.search.openSearch') }}{{ advancedFilterCountDisplay > 0 && `(${advancedFilterCountDisplay})` || ''}}
        </span>
```

Reply:
fixed